### PR TITLE
Adding labels to placeholders (UnknownText, MovementData) in the maps folder

### PR DESCRIPTION
**Pull request by DeeDeeG**
Quick work-in-progress PR for feedback.

Let me know here or in the Discord what you think. Glad to finish this file and other ones I can figure out as I get the time.

### CODE REVIEW

**Comment by mid-kid** [comment]
This looks good. All the advice I can give really is to maybe avoid including unnecessary information in the label's name, to make it shorter. For example `RadioTower1FLuckyNumberManYouHaveAPerfectMatchText` could simply be `RadioTower1FLuckyNumberManMatchText`, and `RadioTower1FReceptionistNotOfferingToursText`  could be `RadioTower1FReceptionistNoToursText`.
There's no limit really to the length of the labels but there's a point where too much information is just not beneficial. Try to condense it into one or a couple of words to simply be able to distinguish the labels and guess at the meaning, if someone wants to read the actual text he can just scroll down.

**Comment by DeeDeeG** [comment]
I was going for the info should have a subject/verb/object, since we're used to reading those a lot IRL and our brain is fast at parsing that kind of info. But it is long.

I probably wouldn't want to do "[...]MatchText", since there's three different levels of matches with the Lucky Number Show. (For 5 digits of ID (perfect match), 3 digits (medium match) and 2 digits (weak match). )

I could see paring it down to "[...]PerfectMatchText", which is a bit shorter than "[...]YouHaveAPerfectMatchText".

But Subject Verb Object is probably too much info/too long?

**Comment by DeeDeeG** [comment]
(Commits aren't in their final state, planning to clean it up a bit when this is ready to merge.)

File: maps/RadioTower1F.asm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-	const_def 2 ; object constants
+RadioTower1FRadioCardWomanOfferQuizText	const_def 2 ; object constants
```
Comment (mid-kid) [inline maps/RadioTower1F.asm]: This doesn't look right.

Comment (DeeDeeG) [reply maps/RadioTower1F.asm]: There's tons of copy-pasting, I must have hit Ctrl+V when I didn't mean to.

Comment (DeeDeeG) [reply maps/RadioTower1F.asm]: Thanks.

**Comment by mid-kid** [comment]
Ah, makes sense. Well, I think such a format would work too, and the point of labels is to be readable to humans, so it's good. Whether you go for Subject/Verb/Object or Verb/Object, however, please try to keep it consistent.

**Comment by DeeDeeG** [comment]
I've been addressing new labels until the file is done, and I plan to circle back to everything in commit one and re-do it.

File: maps/RadioTower1F.asm (hunk 1 above)
Comment (DeeDeeG) [reply maps/RadioTower1F.asm]: (Actually I think I deleted a newline)

Comment (DeeDeeG) [reply maps/RadioTower1F.asm]: Nope, this definitely doesn't belong at the top of the file... Right the first time.

**Comment by DeeDeeG** [comment]
This file is totally done now. :raised_hands:

**Comment by DeeDeeG** [comment]
`RadioTower2F.asm` is done now.

File: maps/Route35NationalParkGate.asm [hunk 2]
```diff
@@ -134,65 +134,65 @@ Route35NationalParkGate_LeaveTheRestBehind:
...
 	ifequal TRUE, Route35NationalParkGate_FirstMonIsEgg
-	writetext UnknownText_0x6a4c6
+	writetext Route35NationalParkGateOfficer1AskToUseFirstMonText
 	yesorno
 	iffalse Route35NationalParkGate_DeclinedToLeaveMonsBehind
 	special ContestDropOffMons
 	iftrue Route35NationalParkGate_FirstMonIsFainted
 	setevent EVENT_LEFT_MONS_WITH_CONTEST_OFFICER
-	writetext UnknownText_0x6a537
+	writetext Route35NationalParkGateOfficer1WellHoldYourMonText
```
Comment (DeeDeeG) [inline maps/Route35NationalParkGate.asm]: Maybe use `WeWillHoldYourMons` instead of `WellHoldYourMons`? (as mentioned on Discord)

**Comment by Rangi42** [comment]
Can this be merged? Are the labels reviewed?

**Comment by DeeDeeG** [comment]
I don't think anyone has gone through and reviewed these yet.

If there's one or a few that  are objectionable, I'd be glad to fix them, or else someone can get around to improving them in a future PR...

If you want any of the commits squashed together or reorganized, I can do that as well.

**Comment by Rangi42** [comment]
Don't worry about squashing, that can be done all at once when it's merged.

**Pull request merged**